contents/WoxelCraft.oscmeta

```diff
@@ -0,0 +1,29 @@
+{
+    "information": {
+        "name": "WoxelCraft",
+        "author": "DaeFennek",
+        "category": "games",
+        "peripherals": ["Wii Remote"],
+        "version": "v0.0.7"
+    },
+    "source": {
+        "type": "github_release",
+        "format": "zip",
+        "repository": "kperdlich/wii-voxel-engine",
+        "file": "woxelcraft_0_0_7.rar"
```

Comment:
format: zip
file: rar

rar files are currently [not supported](https://docs.oscwii.org/repository-manager/supported-archive-formats) by Repository Manager, will look into implementing support and let you know how to proceed with this app.

Reply:
i'll temporarely change the format to `rar` and commit

Reply:
it unfortunately doesn't look like it will be simple to add rar support, remove this app for now.